src/main/java/com/massivecraft/factions/data/MemoryBoard.java

```diff
@@ -272,81 +272,84 @@ public int getFactionCoordCountInWorld(Faction faction, String worldName) {
      * The map is relative to a coord and a faction north is in the direction of decreasing x east is in the direction
      * of decreasing z
      */
-    public ArrayList<FancyMessage> getMap(FPlayer fplayer, FLocation flocation, double inDegrees) {
-        Faction faction = fplayer.getFaction();
-        ArrayList<FancyMessage> ret = new ArrayList<>();
-        Faction factionLoc = getFactionAt(flocation);
-        ret.add(new FancyMessage(FactionsPlugin.getInstance().txt().titleize("(" + flocation.getCoordString() + ") " + factionLoc.getTag(fplayer))));
-
-        // Get the compass
-        ArrayList<String> asciiCompass = AsciiCompass.getAsciiCompass(inDegrees, ChatColor.RED, FactionsPlugin.getInstance().txt().parse("<a>"));
-
-        int halfWidth = FactionsPlugin.getInstance().conf().map().getWidth() / 2;
-        // Use player's value for height
-        int halfHeight = fplayer.getMapHeight() / 2;
-        FLocation topLeft = flocation.getRelative(-halfWidth, -halfHeight);
-        int width = halfWidth * 2 + 1;
-        int height = halfHeight * 2 + 1;
-
-        if (FactionsPlugin.getInstance().conf().map().isShowFactionKey()) {
-            height--;
-        }
+    public ArrayList<FancyMessage> getMap(FPlayer fPlayer, FLocation fLocation, double inDegrees) {
+        final MainConfig conf = FactionsPlugin.getInstance().conf();
+        final ArrayList<FancyMessage> out = new ArrayList<>();
 
-        Map<String, Character> fList = new HashMap<>();
-        int chrIdx = 0;
+        final Faction senderFaction = fPlayer.getFaction();
+        final Faction senderFactionAtLocation = getFactionAt(fLocation);
```

Comment:
Should probably just be 'factionAtLocation` since the faction returned by this function is not associated with the sender.